src/fromager/sources.py

```diff
@@ -38,6 +40,13 @@
 logger = logging.getLogger(__name__)
 
 
+class SDistBackend(enum.StrEnum):
+    # build sdist with PEP 517 hook
+    PEP517 = "pep571"
```

Comment:
🙃 typo, but PEP 571 is the `manylinux2010` PEP so could be a deep feature too

Reply:
ouch :)